kotsadm/web/src/components/modals/DeleteRedactorSpec.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import Modal from "react-modal";
+import moment from "moment";
```

Comment:
We should use `dayjs` instead of moment. All of the date formatters in the utilities file are using it and I don't think we use `moment` anywhere else (at least we shouldn't be). Matter as fact the moment package should be removed entirely.